specification/fabric/Microsoft.Fabric.Management/models.tsp

```diff
@@ -0,0 +1,176 @@
+import "@typespec/rest";
+import "@typespec/http";
+import "@azure-tools/typespec-azure-core";
+
+using TypeSpec.Http;
+using Azure.Core;
+using Azure.ResourceManager;
+using Azure.ResourceManager.Foundations;
+using OpenAPI;
+
+namespace Microsoft.Fabric;
+
+interface Operations extends Azure.ResourceManager.Operations {}
+
+@doc("The Microsoft Fabric capacity properties.")
+model FabricCapacityProperties {
+  @doc("The current deployment state of Microsoft Fabric resource. The provisioningState is to indicate states for resource provisioning.")
+  @visibility("read")
+  provisioningState?: ProvisioningState;
+
+  @doc("The current state of Microsoft Fabric resource. The state is to indicate more states outside of resource provisioning.")
+  @visibility("read")
+  state: State;
+
+  @doc("The capacity administration")
+  administration: CapacityAdministration;
+}
+
+@doc("Represents the SKU name and Azure pricing tier for Microsoft Fabric capacity resource.")
+model RpSku {
+  @doc("The name of the SKU level.")
+  name: string;
+
+  @doc("The name of the Azure pricing tier to which the SKU applies.")
+  tier: RpSkuTier;
+}
+
+@doc("The name of the Azure pricing tier to which the SKU applies.")
+union RpSkuTier {
+  string,
+
+  @doc("Fabric tier")
+  Fabric: "Fabric",
+}
+
+@doc("The administration properties of the Fabric capacity resource")
+model CapacityAdministration {
+  @doc("An array of administrator user identities.")
+  members: Array<string>;
+}
+
+@doc("The resource provisioning state.")
+union ProvisioningState {
+  ResourceProvisioningState,
+
+  /** Resource is deleting */
+  Deleting: "Deleting",
+
+  /** Resource is provisioning */
+  Provisioning: "Provisioning",
```

Comment:
Creating?

Reply:
I wanted to get clarification on the state names. Updating makes sense to me as a transition state for PATCH and PUT of an existing resource. Deleting makes sense as a transition state for DELETE of an existing resource. That means that Provisioning would only happen on PUT of a newly created resource, which is why I think Creating makes is a better state name. Thoughts?

Reply:
agree